### PR TITLE
Update deps for OTP-RR 1.0.0-alpha.x

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
EDIT: Updated for OTP-RR 1.0.0-alpha.x

This PR includes changes involving the full use of OTP-UI library in OTP-RR. Package versions, configurations, and certain UI elements are updated accordingly.

To use, simply execute `yarn start`.
(`yarn install` should automatically happen and update your packages.)

## Old description

~This PR is the first one that is against new branch[`otp-ui-refactor`](https://github.com/ibi-group/trimet-mod-otp/tree/otp-ui-refactor) dealing with OTP-RR refactor.~

~This PR updates the dependencies to just compile `trimet-mod-otp` using OTP-RR 1.0 beta and pass the `yarn run build` stage.~

~To actually run the UI with these dependencies, run `yarn install` on this project, then on the OTP-RR project, run `./cpbuild.sh` with this branch: [`otp-ui-refactor-less-timezone`](https://github.com/opentripplanner/otp-react-redux/tree/otp-ui-refactor-less-timezone). Finally, on this project, run `yarn start`.~

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,17 +17,21 @@
     "core-js": "2",
     "es6-math": "^1.0.0",
     "history": "^4.7.2",
-    "otp-react-redux": "^0.13.7",
+    "leaflet": "^1.6.0",
+    "leaflet.polylinemeasure": "github:ppete2/Leaflet.PolylineMeasure",
```
Comment (landonreed) [inline package.json:21]: Let's not add leaflet polylinemeasure here. Instead let's remove the css import in main.js (?).

Comment (binh-dam-ibigroup) [reply package.json]: Updated in commit 2295153.

**Comment by evansiroky** [comment]
Code seems fine, but when I select one of the bike rental/escooter rental modes, the map doesn't change to show these vehicles. We should figure out why this is happening and make sure otp-react-redux is working this way.

**Comment by binh-dam-ibigroup** [comment]
> Code seems fine, but when I select one of the bike rental/escooter rental modes, the map doesn't change to show these vehicles. We should figure out why this is happening and make sure otp-react-redux is working this way.

@evansiroky your comment above is handled by https://github.com/opentripplanner/otp-react-redux/pull/185.

File: lib/config.yml [hunk 2]
```diff
@@ -538,19 +365,18 @@ errorMessages:
     msg: No available transit routes or rideshare/carshare service at origin.
 
 reportIssue:
-  mailto: gis@trimet.org
+  mailto: hello@trimet.org
 
 analytics:
   google:
-    globalSiteTag: UA-133677388-1
+    globalSiteTag: UA-688646-7
```
Comment (evansiroky) [inline lib/config.yml:372]: I don't think it'd be a good idea to use this google analytics key on our dev site. Can you change it back?

**Comment by evansiroky** [comment]
Ready for final review I believe.

**Pull request merged**